### PR TITLE
[Doc Issue][Update token parameter and link for SSO][3077745]

### PR DESCRIPTION
**Pull request by Namitha-23**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/423a79b55854be2aaae17bb12fb9882d8fcebd1c/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 89 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41b2eca2-7a77-456d-8208-24c8709c5780) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/authentication/tab-sso-code.md [hunk 1]
```diff
@@ -159,8 +159,8 @@ There are a number of libraries available that can handle JWT validation. Basic
 
 Keep in mind the following guidelines when validating the token:
 
-- Valid SSO tokens are issued by Azure AD. The `iss` claim in the token should start with this value.
-- The token's `aud1` parameter will be set to the app ID generated during Azure AD app registration.
+- Valid SSO tokens are issued by Azure AD. The `iss` claim in the token should start with this [value](https://login.microsoftonline.com/).
```
Comment (yuezk) [inline msteams-platform/tabs/how-to/authentication/tab-sso-code.md:162]: From my limited understanding, I'm afraid `value` might not be a link.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64e899a3f5757c1e7fb39175d712dea1a3cd77f6/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87c9bb87-d2bf-499a-9074-e96d23860437) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/839fdf507ae46057fe929fc35bf1cbb928724fb2/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63c41926-6e8f-4409-95f7-401dc87ff888) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3067708fa393e66bbc56ef818fe8e1707e657048/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8b395af-07d0-4c76-b49b-1a9f98ea451b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61b4757505782709f97b26fdb21e8e69fd9215d6/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0050d934-4636-4d7c-87cd-f427d5e889f8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/App-permissions/Teams-app-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7f7041711ef98df0dc32bc69d31a2e9544ca470e/msteams-platform%2Fgraph-api%2FApp-permissions%2FTeams-app-permissions.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cdcb932e-a973-4c2f-a7aa-d577d81acddc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7f7041711ef98df0dc32bc69d31a2e9544ca470e/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/715194df-1213-40a9-aece-ffe23e296de3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Nivedipa-MSFT — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/App-permissions/Teams-app-permissions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3d60b4e7776c48100be009c9bd0d430fab30d44/msteams-platform%2Fgraph-api%2FApp-permissions%2FTeams-app-permissions.md) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54fae3ab-f8d2-418c-a7b2-52e6df0b1b8a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/authentication/tab-sso-code.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3d60b4e7776c48100be009c9bd0d430fab30d44/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-code.md) | 91 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c75adf5-f664-4193-a873-b2561ce1f202) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by Namitha-23** [comment]
The PR has been approved: 
MicrosoftTeams-image (4)

**Comment by Namitha-23** [comment]
Closing this PR as the content has been added to https://github.com/MicrosoftDocs/msteams-docs/pull/8716 .

**Pull request closed without merging**